plugin/external/pluginEntry.go

```diff
@@ -370,15 +451,13 @@ func (e *pluginEntry) List(ctx context.Context) ([]plugin.Entry, error) {
 		entry.schemaGraphs = e.schemaGraphs
 		entries[i] = entry
 	}
+
 	return entries, nil
 }
 
 func (e *pluginEntry) Read(ctx context.Context) ([]byte, error) {
-	if impl := e.methods["read"].result; impl != nil {
-		if content, ok := impl.(string); ok {
-			return []byte(content), nil
-		}
-		return nil, fmt.Errorf("Read method must provide a string, not %v", impl)
+	if impl := e.methods["read"].tupleValue; impl != nil {
```

Comment:
The tupleValue should already be parsed by this point right? If so, isn't it enough to just cast to content or byte or whatever we're returning?

Reply:
It is now.